site/docs/resources/migrating_to_0_10_0.md

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_position: 3
+---
+
+# Migrating to 0.10.0 ⛷️
```

Comment:
```suggestion
# Syntax changes in 0.10.0 ⛷️
```
I would prefer to stay away from "migration" since that makes me think a lot of work will need to be done on the part of the devs. Syntax change also feels a bit more precise to me.